cmd/rule-evaluator/main.go

```diff
@@ -542,8 +583,13 @@ func sendAlerts(s *notifier.Manager, projectID string, externalURL *url.URL) rul
 			}
 			if externalURL != nil {
 				if externalURL.String() == googleCloudBaseURL.String() {
```

Comment:
Want to rename `externalURL` with `generatorURL` (this is what we get from callers)